src/simput/types/nwchem-neb/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+type: 'nwchem-neb',
```

Comment:
ESLint complains about this file, on indentation and missing semi-colon, but they're non-consequential.